### ISSUE TITLE
[Bug]: chunk_bwd_dv_local 精度测试在 T>1 场景全量失败

### ISSUE THREAD

**Issue by SEU-ZJY** [flashserve/flash-linear-attention-npu#456]
### 环境信息

python version: 3.9.25
CANN version: version=26.1.0
CANN public version: 9.2.0
torch version: 2.7.1+cpu
torch_npu version: 2.7.1.post6.dev20260723
triton-ascend version: 3.2.0
ascend：910B

### 最小复现
chunk_bwd_dv_local 在输入 g 非单调时可以稳定复现 NPU 与 标杆精度不一致。
算子底层 kernel 对 gate factor 的处理逻辑为 g_factor = exp(min(g_j - g_i, 0))，但标杆中的处理逻辑是 g_factor = torch.exp(g_j - g_i)。因此，只要随机生成的 g 存在 g_j > g_i，就会出现“标杆中 exp(g_j - g_i) > 1，NPU kernel 中 exp(min(g_j - g_i, 0)) = 1”，从而产生精度差异。

例如固定长度场景：
```text
q:             fp16 [1, 1, 64, 128]
k:             fp16 [1, 1, 64, 128]
d_o:           fp16 [1, 1, 64, 128]
g:             fp16 [1, 1, 64]
g range:       [-1.0, 0.0]，逐元素随机生成，无法保证沿 T 单调
scale:         1 / sqrt(128)
chunk_size:    64
g_gamma:       None
A:             None
cu_seqlens:    None
chunk_indices: None
```
算子本身能够正常执行，但在 ATK 双标杆精度比较中未通过。

变长模式下同样可以复现，例如：
```text
q:             fp16 [1, 1, 128, 128]
k:             fp16 [1, 1, 128, 128]
d_o:           fp16 [1, 1, 128, 128]
g:             fp16 [1, 1, 128]
g range:       [-1.0, 0.0]，逐元素随机生成
scale:         1 / sqrt(128)
chunk_size:    64
g_gamma:       None
A:             None
cu_seqlens:    [0, 64, 128]
chunk_indices: [[0, 0], [1, 0]]
```
该场景同样能够完成执行，但最终未通过精度测试。

### 完整报错
在 ATK 双标杆精度比较阶段失败：accuracy = False，算子执行本身无报错，NPU 输出能够正常返回。

### 期望行为
希望确认 chunk_bwd_dv_local 对输入 g 的正式语义约束。
如果算子接口允许 g 为合法范围内的任意值，那么标杆应该与 kernel 保持一致。
如果算子实际上要求输入 g 必须满足g_j - g_i <= 0，那么希望在算子文档中明确说明该约束。

**Comment by chen-linxin** [comment]
重新核对 fla-org 后确认：`chunk_bwd_dv_local` 接收的是 gate activation 做 chunk-local cumsum 后的累计 `g`。gate activation 为 `-exp(A_log) * softplus(...)`，因此不大于 0；累计后的 `g` 在每个序列的每个有效 chunk 内沿 T 维单调不增，序列边界和 chunk 边界重新累计。

Issue 用例直接构造了非单调 `g`，违反了该输入契约。正差值下，Kernel 的防御性截断与 benchmark 的原始指数计算产生差异，从而导致精度失败；这不表示算子需要支持非单调 `g`。

本次修改在 README 中明确累计 gate 与单调约束，golden 与 Kernel 保留正差值截断作为防御性处理。不新增独立测试文件，不修改基础 CI，也不新增 `ct` 导入或依赖；NPU 精度继续由仓库现有算子测试和 ATK 用例验证。

当前验证：Python 语法检查、ATK JSON 解析和 diff 检查通过；更新后的 A2 NPU 与 ATK 用例将在新 head 上重跑，A3/A5 与整体 Example ST 由当前 head 的 NPU CI 补齐。

PR：https://github.com/flashserve/flash-linear-attention-npu/pull/466
